runtime/common/src/migrations.rs

```diff
@@ -154,19 +156,89 @@ where
 	}
 }
 
-pub struct ReferendaMigrations<Runtime, Council, Tech>(PhantomData<(Runtime, Council, Tech)>);
+pub struct MissingBalancesMigrations<T>(PhantomData<T>);
+impl<T> Migration for MissingBalancesMigrations<T>
+where
+	T: pallet_balances::Config,
+	<T as frame_system::Config>::AccountId: Default,
+{
+	fn friendly_name(&self) -> &str {
+		"MM_MissingBalancesMigrations"
+	}
+
+	fn migrate(&self, _available_weight: Weight) -> Weight {
+		pallet_balances::migration::MigrateToTrackInactive::<T, ()>::on_runtime_upgrade();
+		pallet_balances::migration::ResetInactive::<T, ()>::on_runtime_upgrade();
+		pallet_balances::migration::MigrateToTrackInactive::<T, ()>::on_runtime_upgrade()
```

Comment:
Shouldn't this add the weights of all the migrations? Something like
```suggestion
		let w1 = pallet_balances::migration::MigrateToTrackInactive::<T, ()>::on_runtime_upgrade();
		let w2 = pallet_balances::migration::ResetInactive::<T, ()>::on_runtime_upgrade();
		let w3 = pallet_balances::migration::MigrateToTrackInactive::<T, ()>::on_runtime_upgrade();
		
		w1.saturating_add(w2).saturating_add(w3)
```

Reply:
Given that user tx are forbidden during migrations, and that migrations are mandatory, whatever their weight, they are executed in full anyway, precise weight measurement will only be useful when we will perform multi-block migrations (if they ever are).